community/lua-bitop/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor:
+# Maintainer:
```

Comment:
Why no maintainer ?

Reply:
I'd rather not put myself as maintainer for that package as I have little interest in lua and probably don't know how to fix any problems if they come up...